app/xds_api/views.py

```diff
@@ -358,3 +374,114 @@ def add_course_to_lists(request, exp_hash):
     else:
         return Response({"message": "course successfully added!"},
                         status.HTTP_200_OK)
+
+
+@api_view(['GET'])
+def interest_lists_owned(request):
+    """Handles HTTP requests for interest lists managed by request user"""
+    errorMsg = {
+        "message": "Error fetching records please check the logs."
+    }
+    user = request.user
+
+    if not request.user.is_authenticated:
+        return Response({'Please login to view Interest Lists'},
```

Comment:
Since this is an API and not a UI, we might want to be more descriptive about what the issue is.  For example we could say something like "Access Denied: Unauthenticated user."